index.html

```diff
@@ -0,0 +1,176 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Capstone Project</title>
+  </head>
+  <body>
+     <section class="wrapper">
+      <header>
+        <div class="nav-bar1">
+          
+            <a href="facebook.com/paulmalongo"><i class="fa-brands fa-facebook-f"></i></a>
+            <a href="facebook.com/paulmalongo"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="https://canon20paul.github.io/My-Portfolio/">My Page</a>
+            <a href="#">Logout</a>
+          </div>
+        
+        <nav class="nav-bar">
+
+          
+        <ul class="nav-menu">
+          <li class="nav-item logo">
+            
+            <a href="#"><img class="logo-img" src="images/R.png" alt="logo"/> 
+            <h3>REMOTE GLOBAL  SUMMIT<br>2022</h3></a>
+            
+          </li>
+            <li class="nav-item">
+                <a href="" class="nav-link">Home</a>
+            </li>
+            <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+            </li>
+            <li class="nav-item">
+                <a href="" class="nav-link">Program</a>
+            </li>
+            <li class="nav-item">
+                <a href="" class="nav-link">Join</a>
+            </li>
+               <li class="nav-item labelT">
+              <a href="" class="nav-link">WFH Remoters..</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+            
+        </div>
+        </nav>
+    </header>
+        <section class="mainS"></section>
+           <div class="main-header">
+          <p class="hello">"Hello! Am a WFH Work from Home"</p>
+          <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL <br>
+                        GLOBAL &nbsp;SUMMIT <br>2022 
+          </h1>
+          <p class="desc">A joyful celebration believing in the value of opennes and
+             sharing, creating a positive change with people from over 100 countries 
+             is taking place in October, in Kampala, Uganda E.Africa</p>
+             <h2 class="date">2022.10.15(MON ~ 17(WED)</h2>
+             <p class="desc2">@Munyonyo Commonwealth Resort, Along Lake Victoria Shores</p>
+             <!-- <div class="header-icons"> <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+              <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+              <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+             </div> -->
+        </div>
+      
+
```

Comment:
- [ ] As I can see the alignment for the home page header section is not like the original design so kindly work on that to make the alignment be the same I can show you the difference between both 


- Your design

![image](https://user-images.githubusercontent.com/58553711/183261928-75135b47-7881-4989-9192-326e487d081a.png)


- Original design
![image](https://user-images.githubusercontent.com/58553711/183261911-9f156879-5e08-4ee0-a9ec-4e508bf7e5bc.png)